astronomer/providers/amazon/aws/example_dags/example_delete_eks_cluster_and_role_policies.sh

```diff
@@ -77,5 +77,5 @@ fi
 
 # Command to delete the EKS cluster and node group attached with it. Make sure to wait for the cluster to be deleted so
 # that the task reports for any potential failures.
-eksctl delete cluster $EKS_CLUSTER_NAME --wait
+eksctl delete cluster $EKS_CLUSTER_NAME --wait --force --timeout 60m0s
```

Comment:
```suggestion
eksctl delete cluster $EKS_CLUSTER_NAME --wait --force --timeout 600ms
```

Reply:
The default value is 25 minutes 0 seconds. The syntax is the form X minutes Y seconds -> XmYs

Reply:
<img width="1550" alt="Screenshot 2023-06-05 at 5 50 18 PM" src="https://github.com/astronomer/astronomer-providers/assets/10206082/4e163d2b-45c8-419f-9eed-9e440c1ebe2f">